mlrun/config.py

```diff
@@ -1115,10 +1115,22 @@ def get_model_monitoring_file_target_path(
                     if function_name is None
                     else f"{kind}-{function_name.lower()}",
                 )
-            return mlrun.mlconf.model_endpoint_monitoring.store_prefixes.default.format(
-                project=project,
-                kind=kind,
-            )
+            elif kind == "stream":  # ML-6043
```

Comment:
```suggestion
            elif kind == "stream":  # ML-6043
```
I suggest to remove the `#ML-6043` notes and explain here that we return 2 stream paths in order to handle backwards compatibility of old mlrun versions (<1.7)